db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -30,51 +30,79 @@ class NonBatchedOpsStressTest : public StressTest {
       ts = ts_str;
       options.timestamp = &ts;
     }
+
     auto shared = thread->shared;
     const int64_t max_key = shared->GetMaxKey();
     const int64_t keys_per_thread = max_key / shared->GetNumThreads();
     int64_t start = keys_per_thread * thread->tid;
     int64_t end = start + keys_per_thread;
     uint64_t prefix_to_use =
         (FLAGS_prefix_size < 0) ? 1 : static_cast<size_t>(FLAGS_prefix_size);
+
     if (thread->tid == shared->GetNumThreads() - 1) {
       end = max_key;
     }
+
     for (size_t cf = 0; cf < column_families_.size(); ++cf) {
       if (thread->shared->HasVerificationFailedYet()) {
         break;
       }
-      if (thread->rand.OneIn(4)) {
-        // 1/4 chance use iterator to verify this range
-        Slice prefix;
-        std::string seek_key = Key(start);
+
+      enum class VerificationMethod {
+        Iterator,
+        Get,
+        MultiGet,
+        GetMergeOperands,
+        NumberOfMethods
```

Comment:
Nit: maybe add a note so that future developers will not accidentally any item below `kNumberOfMethods`.
Also, I think our naming convention prefers `kIterator`, etc.

Reply:
Will do. Thanks for the quick review @riversand963 !